### ISSUE TITLE
An option to delete a workflow run

### ISSUE THREAD

**Issue by vazome** [github/vscode-github-actions#293]
**Is your feature request related to a problem? Please describe.**
I think it would be great if we could delete a workflow run, the same way we can delete a variable/secret from the VS Code.

**Describe the solution you'd like**
A delete button to the right of the run.
<img width="504" alt="image" src="https://github.com/github/vscode-github-actions/assets/46573198/2d498a55-0b1d-4653-a1ea-7bb35a43291f">
<img width="59" alt="image" src="https://github.com/github/vscode-github-actions/assets/46573198/f980faff-c3c8-4bfd-af02-9756a0b5fd24">

**Additional context**
None

**Comment by felipesu19** [comment]
Agree! Like i've said in other issues, we're not doing any active feature development, but i would welcome a PR!

**Comment by ryux1** [comment]
I’m working on this. I’ll add a confirmed delete action for completed workflow runs, expose it only when the repository permission permits the operation, refresh both run views after deletion, and cover the new run context state with tests.
